README.md

```diff
@@ -26,15 +26,62 @@ It's built with [Hugo](https://gohugo.io/) and hosted at https://iceberg.apache.
 
 The Iceberg documentation site is actually constructed from two hugo sites. The first, is the landing page. The second site, 
 is the documentation site which contains the full Iceberg documentation, including the javadoc. The landing page and
-documentation sites are completely self-contained in the `./landing-page` and `./docs` directories, respectively.
+documentation sites are completely self-contained in the `./landing-page` and `./docs` directories, respectively. The Javadocs are in the `./javadoc` directory.
 
-# Landing Page Deployment
```

Comment:
I moved up contribution guide because readers are most likely interested in that